L/libjulia/common.jl

```diff
@@ -0,0 +1,161 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg.Types
+
+# Collection of sources required to build GMP
+function configure(version)
+    name = "libjulia"
+
+    checksums = Dict(
+        v"1.4.2" => "76a94e06e68fb99822e0876a37c2ed3873e9061e895ab826fd8c9fc7e2f52795",
+        v"1.5.1" => "1f138205772eb1e565f1d7ccd6f237be8a4d18713a3466e3b8d3a6aad6483fd9",
+    )
+    sources = [
+        ArchiveSource("https://github.com/JuliaLang/julia/releases/download/v$(version)/julia-$(version).tar.gz", checksums[version]),
+        DirectorySource("./bundled"),
+    ]
+
+    # Bash recipe for building across all platforms
+    script = raw"""
+    apk update
+    apk add coreutils libuv-dev utf8proc
+
+    cd $WORKSPACE/srcdir/julia*
+
+    # Apply patches
+    if [ -d $WORKSPACE/srcdir/patches ]; then
+    for f in $WORKSPACE/srcdir/patches/*.patch; do
+        echo "Applying path ${f}"
+        atomic_patch -p1 ${f}
+    done
+    fi
+
+    case ${target} in
+        *linux*)
+            OS=Linux
+        ;;
+        *mingw*)
+            OS=WINNT
+        ;;
+        *darwin*)
+            OS=Darwin
+        ;;
+        *freebsd*)
+            OS=FreeBSD
+        ;;
+    esac
+
+    cat << EOM >Make.host.user
+    override CC=${CC_BUILD}
+    override CXX=${CXX_BUILD}
+    override AR=${AR_BUILD}
+    USE_SYSTEM_LIBUV=1
+    USE_SYSTEM_UTF8PROC=1
+    # julia want's libuv.a
+    override LIBUV=/usr/lib/libuv.so
+    override LIBUTF8PROC=/usr/lib/libutf8proc.so.2
+    EOM
+
+    cat << EOM >Make.user
+    USE_SYSTEM_LLVM=1
+    USE_SYSTEM_LIBUNWIND=1
+    USE_SYSTEM_PCRE=1
+    USE_SYSTEM_OPENLIBM=1
+    USE_SYSTEM_DSFMT=1
+    USE_SYSTEM_BLAS=1
+    LIBBLASNAME=libopenblas
+    USE_SYSTEM_LAPACK=1
+    LIBLAPACKNAME=libopenblas
+    USE_SYSTEM_GMP=1
+    USE_SYSTEM_MPFR=1
+    USE_SYSTEM_SUITESPARSE=1
+    USE_SYSTEM_LIBUV=1
+    USE_SYSTEM_UTF8PROC=1
+    USE_SYSTEM_MBEDTLS=1
+    USE_SYSTEM_LIBSSH2=1
+    USE_SYSTEM_CURL=1
+    USE_SYSTEM_LIBGIT2=1
+    USE_SYSTEM_PATCHELF=1
+    USE_SYSTEM_ZLIB=1
+    USE_SYSTEM_P7ZIP=1
+
+    override XC_HOST=${target}
+    override OS=${OS}
+
+    #llvm-config-host is not available
+    override LLVMLINK=-L${prefix}/lib -lLLVM-9jl
+    override LLVM_CXXFLAGS=-I${prefix}/include # This is probably to simple
+    override LLVM_LDFLAGS=-L${prefix}/lib
+
+    # just nop this
+    override LLVM_CONFIG_HOST=
```

Comment:
We should use the `llvm-config` from the `x86_64-linux-musl` build of LLVM_full, as our `LLVM_CONFIG_HOST`. 
But that was more work than just hardcoding.